scripts/publish.ts

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env -S node_modules/.bin/tsx
+// https://github.com/google/zx/issues/467#issuecomment-1577838056
+
+import { $ } from 'zx'
+import fs from 'fs/promises'
+import semver from 'semver'
+
+const packageFolder = process.argv[3] ?? ''
```

Comment:
```suggestion
const packageFolder = process.argv[2] ?? ''
```

I think here it would be the third argument, right?